datasources.tsv

```diff
@@ -139,3 +139,6 @@ Wikipedia	Wi	http://www.wikipedia.org	https://en.wikipedia.org/wiki/$id	Acetate
 WormBase	W	http://www.wormbase.org/	http://www.wormbase.org/db/gene/gene?name=$id;class=Gene	WBGene00000001	gene	Caenorhabditis elegans	1	urn:miriam:wb	^WBGene\d{8}$	WormBase	P3860	wormbase	gene
 XMetDB	Xm	http://www.xmetdb.org/	http://www.xmetdb.org/xmetdb/protocol/$id	XMETDB171	interaction		1		^XMETDB\d+$	Xenobiotics Metabolism Database		xmetdb	interaction
 ZFIN	Z	http://zfin.org	http://zfin.org/action/marker/view/$id	ZDB-GENE-041118-11	gene	Danio rerio	1	urn:miriam:zfin	ZDB\-GENE\-\d+\-\d+	ZFIN Gene	P3870	zfin	gene
+Github	github	https://github.com	https://github.com/$id	githubuser	author		0	urn:miriam:github	^[a-zA-Z0-9-_]+(/[a-zA-Z0-9-_]+)?$	Github		github	author
```

Comment:
Plz change to GitHub in the first column and in the later column near the end.

Reply:
In the example identifier, use something non-random, plz. Who knows what may show up there in the future.

Reply:
Thank you, made the changes and squashed the commits into 1 commit.